server/models/Share.ts

```diff
@@ -188,10 +192,11 @@ class Share extends IdModel<
   @Column
   allowIndexing: boolean;
 
-  revoke(userId: string) {
+  revoke(ctx: APIContext, eventOverride?: EventOverride) {
```

Comment:
No need to allow passing `eventOverride` here, we can just hardcode it on L199?

Reply:
Yeah 🤦‍♂️